bin/vm-agent.js

```diff
@@ -98,11 +95,13 @@ function loadSysinfo(callback) {
 }
 
 
-async.waterfall([
+vasync.pipeline({funcs: [
```

Comment:
This is a good point. While doing this I also found that there's a bug here in the loadConfig/loadSdcConfig logic. I'll fix shortly.